railties/lib/rails/generators/rails/db/system/change/change_generator.rb

```diff
@@ -13,6 +13,9 @@ class ChangeGenerator < Base # :nodoc:
           class_option :to, required: true,
             desc: "The database system to switch to."
 
+          class_option :skip_devcontainer, type: :boolean, default: true,
```

Comment:
```suggestion
          class_option :devcontainer, type: :boolean, default: false,
```

This would give us `--no-devcontainer`. We should do the same on the app generator as well

Reply:
TIL :D